src/core/state/state.py

```diff
@@ -40,6 +40,7 @@ class State:
 
     force_propulsion_thrusters: float = 0.0
     fuel_mass: float = 0.0
+    dry_mass: float = 0.0
```

Comment:
I thought it would be useful for the integrator when updating the position, velocity, and angular velocity to know the current mass of the cubesat (dry mass + fuel mass)

Reply:
Ah I see. The reason I asked is because the dry mass of the spacecraft really should not change throughout the mission (unless you're simulating some truly bizarre mission cases, like having a micrometeoroid cleave off some of the spacecraft's brackets) so having it in state.py seemed a bit strange to me at first since every variable in there is constantly changing. However, even if it's not changing, the dry mass is still a fundamental state of the system so I think it should stay here.